Chess-FromZeroToHero.DataAccess/Entities/BaseEntity.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chess_FromZeroToHero.DataAccess.Entities
+{
+    public abstract class BaseEntity
+    {
+        public int Id { get; set; }
```

Comment:
Generally it is better to use GUID/UUID for primary key as it gives global uniqueness and mild security by obscurity